pkg/config/config.go

```diff
@@ -222,6 +229,10 @@ func (cfg *Config) Validate() error {
 	if cfg.EnableWebhookServer && cfg.WebhookServerAddr == "" {
 		return errors.New("empty webhook server address")
 	}
+
+	if cfg.DeletionPolicy == "" {
+		cfg.DeletionPolicy = ackv1alpha1.DeletionPolicyDelete
+	}
```

Comment:
This is not the best place to do defaulting, maybe use `flag.StringVar` with a default value?

Reply:
I understand that maybe you want to avoid doing type casting during reconciliation.. maybe we can make DeletionPolicy part of reconciler struct?